### PR TITLE
PICARD-1781: Have $find return '' rather than '-1' on not found

### PR DESCRIPTION
**Pull request by rdswift**
# Summary

* This is a…
    * [ ] Bug fix
    * [ ] Feature addition
    * [ ] Refactoring
    * [ ] Minor / simple change (like a typo)
    * [x] Other
* **Describe this change in 1-2 sentences**: Have `$find` return "" rather than "-1" on not found

# Problem

* JIRA ticket: PICARD-1781

The `$find` scripting function currently returns "-1" if `needle` is not found in `haystack`.  This was based on the return value of the Python `find` method.  This is not consistent with other Picard scripting functions, which return "" for a `False` (not found?) condition.  For consistency, consider changing the `$find` function to return "" on a "not found" condition.

# Solution

Have $find return "" rather than "-1" on not found.

# Action

### CODE REVIEW

File: picard/script.py [hunk 1]
```diff
@@ -1047,7 +1047,10 @@ def func_find(parser, haystack, needle):
     Returns:
         The zero-based index of the first occurrance of needle in haystack, or -1 if needle was not found.
```
Comment (zas) [inline picard/script.py]: s/occurance/occurence/?

Comment (phw) [reply picard/script.py]: The docstring needs to be updated to reflect the change

**Comment by rdswift** [comment]
Will the next release be 2.3.2 so I can make a note in the docs about the response change taking effect at a certain version?  Otherwise, I'll make a note about it being "-1" up to Picard 2.3.1 and "" after.  Will there be a target branch set up in picard-website to capture the changes in documentation until the new version is released?  Thanks.

**Comment by phw** [comment]
@rdswift I guess we'll have a 2.3.2 and afterwards concentrate on the 2.4. So I have setup a https://github.com/metabrainz/picard-website/tree/picard-2.3.2 .

**Pull request merged**